x/posts/internal/keeper/keeper_test.go

```diff
@@ -75,6 +78,16 @@ func TestKeeper_SavePost(t *testing.T) {
 			},
 			newPost:              types.NewPost(types.PostID(15), types.PostID(1), "Comment", false, "", 0, testPostOwner),
 			expParentCommentsIDs: []types.PostID{types.PostID(15)},
+			expLastID:            types.PostID(15),
+		},
+		{
+			name: "Post with ID greater ID than Last ID stored",
```

Comment:
Perfect. 

Can we have another test case where we save a Post that has in ID that is **not** greater than the last one? This should check that the last one stays the greater one without being updated to match the smallest.

Reply:
ofc